src/main/java/com/googlecode/jsonrpc4j/spring/JsonProxyFactoryBean.java

```diff
@@ -54,38 +54,40 @@ public void afterPropertiesSet() {
 		super.afterPropertiesSet();
 		proxyObject = ProxyFactory.getProxy(getServiceInterface(), this);
 
-		if (jsonRpcHttpClient==null) {
-			if (objectMapper == null && applicationContext != null && applicationContext.containsBean("objectMapper")) {
-				objectMapper = (ObjectMapper) applicationContext.getBean("objectMapper");
-			}
-			if (objectMapper == null && applicationContext != null) {
-				try {
-					objectMapper = BeanFactoryUtils.beanOfTypeIncludingAncestors(applicationContext, ObjectMapper.class);
-				} catch (Exception e) {
-					logger.debug(e);
-				}
+		objectMapper = null;
```

Comment:
Same question here - what's the purpose of removing the null check and setting `objectMapper` to `null`?